maintainers/maintainer-list.nix

```
@@ -10117,6 +10117,12 @@
     githubId = 36448130;
     name = "Michael Brantley";
   };
+  linbreux = {
+    email = "linbreux@gmail.com";
+    github = "linbreux";
```

```suggestion
    github = "Linbreux";
```

according to https://api.github.com/user/29354411